test/unit/translators/twitter_translators/tweet_translator_test.rb

```diff
@@ -16,28 +16,28 @@ def build_tweet
     twitter_user = build_twitter_user
 
     OpenStruct.new(
-      id: "my_twitter_noise_id",
-      text: "my_twitter_noise_text",
-      full_text: "my_twitter_noise_text",
+      id: 'my_twitter_noise_id',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.